GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs

```diff
@@ -17,6 +17,7 @@ internal sealed class QuickSearchProvider
         private readonly RevisionDataGridView _gridView;
         private readonly Timer _quickSearchTimer;
         private readonly IGitRevisionTester _gitRevisionTester;
+        private readonly string _initialString = "Searching for: ";
```

Comment:
This string must be made localisable. 
Please have a look at the `Strings` class, and add this string there.

Reply:
Done